tests/basic/storage_pool_test.go

```diff
@@ -9762,3 +9762,106 @@ var _ = Describe("{AddDriveMetadataPool}", func() {
+		log.InfoD("Pool UUID on which IO is running [%s]", poolUUID)
+
+		// Get Node Details of the Pool with IO
+		nodeDetail, err := GetNodeWithGivenPoolID(poolUUID)
+		log.FailOnError(err, "Failed to get Node Details from PoolUUID [%v]", poolUUID)
+		log.InfoD("Pool with UUID [%v] present in Node [%v]", poolUUID, nodeDetail.Name)
+
+		if IsLocalCluster(*nodeDetail) != true {
+			log.FailOnError(fmt.Errorf("This test will only support onprem vms"), "is this onprem?")
+		}
+		// Get Total Pools present on the Node present
+		poolDetails, err := GetPoolsDetailsOnNode(*nodeDetail)
+		log.FailOnError(err, "Failed to get Pool Details from Node [%v]", nodeDetail.Name)
+		log.InfoD("List of Pools present in the node [%v]", poolDetails)
+		poolToBeResized, err := GetStoragePoolByUUID(poolDetails[0].Uuid)
+		drvSize, err := getPoolDiskSize(poolToBeResized)
+		log.FailOnError(err, "error getting drive size for pool [%s]", poolToBeResized)
+		//add more disk since it should take longer for the resize to take longer
+		driveSize := drvSize * 3
+		expectedSize := (poolToBeResized.TotalSize / units.GiB) + driveSize
+		err = Inst().V.ExpandPoolUsingPxctlCmd(*nodeDetail, poolToBeResized.Uuid, api.SdkStoragePool_RESIZE_TYPE_ADD_DISK, expectedSize, true)
+		log.FailOnError(err, "error getting drive size for pool [%s]", poolToBeResized)
+		expandedPool, err := GetStoragePoolByUUID(poolToBeResized.Uuid)
+		log.FailOnError(err, "error getting pool by using id %s", poolToBeResized.Uuid)
+		dash.VerifyFatal(expandedPool == nil, false, "expanded pool value is nil")
+		err = WaitForExpansionToStart(poolToBeResized.Uuid)
+		if err != nil {
+			log.FailOnError(err, "error getting pool by using id %s", poolToBeResized.Uuid)
+		}
+		var connect node.ConnectionOpts
+		connect.Timeout = 60
+		connect.TimeBeforeRetry = 10
+		err = Inst().N.ShutdownNode(*nodeDetail, node.ShutdownNodeOpts{
+			Force:          true,
+			ConnectionOpts: connect,
+		})
+		log.FailOnError(err, "failed to shutdown the node with err %s", err)
+		time.Sleep(300 * time.Second)
+		log.InfoD("sleeping for 5 mins to wait for shutdown to be completed")
+		t := func() (interface{}, bool, error) {
+			err = Inst().N.PowerOnVM(*nodeDetail)
+			if err != nil {
+				return nil, false, err
+			}
+			return nil, false, nil
+		}
+
+		_, err = task.DoRetryWithTimeout(t, 5*time.Minute, 10*time.Second)
+		log.FailOnError(err, "Failed to powered on the vm")
+		isjournal, err := isJournalEnabled()
+		log.FailOnError(err, "Failed to check if Journal enabled")
+		validatePXStartTimeout := 15 * time.Minute
+		if err := Inst().V.WaitDriverUpOnNode(*nodeDetail, validatePXStartTimeout); err != nil {
+			log.FailOnError(err, "failed to shutdown the node with err %s", err)
+		}
+		poolStatus, err := getPoolLastOperation(expandedPool.Uuid)
+		dash.VerifyFatal(poolStatus.Status != api.SdkStoragePool_OPERATION_FAILED, true, fmt.Sprintf("PoolResize is successful on pool %s", expandedPool.Uuid))
```

Comment:
can you also print the pool status once last operation ?

Reply:
done

Reply:
where are you printing this. also it would be help if message is also printed in case of failed opertion

Reply:
log.InfoD("after poweron the operation status is %v", poolStatus.Status) this is what is the status we are printing after the node is shutdown and powered on

Reply:
it will not go till that line if it fails in this step.